.github/workflows/trivy-scans.yml

```diff
@@ -0,0 +1,61 @@
+name: trivy scans
```

Comment:
If these vulnerability scans are required, update the scan configurations below specific for this project.